doc/developer/frr-release-procedure.rst

```diff
@@ -0,0 +1,181 @@
+.. _frr-release-procedure:
+
+FRR Release Procedure
+=====================
+
+``<version>`` - version to be released, e.g. 7.3
+``origin`` - FRR upstream repository
+
+1. Checkout ``dev/<version>``.
+
+   .. code-block:: shell
+   
+      git checkout dev/<version>
+
+2. Create and push a new branch called ``stable/<version>`` based on the
+   ``dev/<version>`` branch.
+
+   .. code-block:: shell
+
+      git checkout -b stable/<version>
+      git push origin stable/<version>:refs/heads/stable/<version>
+
+3. Update Changelog for Red Hat Packages:
+
+   Edit :file:`redhat/frr.spec.in` and look for the ``%changelog`` section:
+
+   - Change last (top of list) entry from ``%{version}`` to the **last**
+     released version number. For example, if ``<version>`` is ``7.3`` and the
+     last public release was ``7.2``, you would use ``7.2``, changing the file
+     like so::
+
+        * Tue Nov  7 2017 Martin Winter <mwinter@opensourcerouting.org> - %{version}
+
+     to::
+
+        * Tue Nov  7 2017 Martin Winter <mwinter@opensourcerouting.org> - 7.2
+
+   - Add new entry to the top of the list with ``%{version}`` tag.  Make sure
+     to watch the format, i.e. the day is always 2 characters, with the 1st
+     character being a space if the day is one digit.
```

Comment:
maybe we can use `rpmbuild` to verify spec file as we do for Debian packages (`dpkg-parsechangelog`)?

Reply:
I don't know how to use that, feel free to add another commit documenting it.